Quadriga/src/main/java/edu/asu/spring/quadriga/dto/AboutTextDTO.java

```diff
@@ -0,0 +1,70 @@
+package edu.asu.spring.quadriga.dto;
+
+import java.io.Serializable;
+
+import javax.persistence.Basic;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+/**
+ * DTO for about project text of project's public website.
+ * 
+ * @author Rajat Aggarwal
+ */
+
+@Entity
+@Table(name = "tbl_about_text")
+public class AboutTextDTO implements Serializable {
+
+    private static final long serialVersionUID = 1103019135158917211L;
+
+    @Id
+    private String Id;
+
+    public String getId() {
+        return Id;
+    }
+
+    public void setId(String id) {
+        Id = id;
+    }
+
+    @Basic(optional = false)
+    @Column(name = "ProjectId")
+    private String projectId;
+
+    @Basic(optional = false)
+    @Column(name = "title")
+    private String title;
+
+    @Basic(optional = false)
```

Comment:
I wouldn't put these restrictions on the database level. You can check that there is a title/description in the validation of the form, but I wouldn't put it here. This might be something we'd like to make optional in the future. Even with projectId I would be careful. It makes it more difficult to reuse in the future, if we put those kind of restrictions on the database level.